### PR TITLE
Emagged calculators can now spawn Baldi

### PR DESCRIPTION
**Pull request by BobJoga**
## About The Pull Request

title
emagged calculators will now spawn a baldi mob if three problems are answered incorrectly, one after the other. this can spawn two baldis max per calculator.
baldi mobs can only be killed by answering three problems correctly in a row on an emagged calculator.

## Why It's Good For The Game

honestly i don't even know how i could possibly justify this

## Changelog
:cl:
add: Emagged calculators can now spawn Baldi.
imageadd: Baldi icons.
/:cl:

### CODE REVIEW

File: modular_skyrat/code/modules/mob/living/simple_animal/hostile/baldi.dm [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	melee_damage_upper = 65
+	melee_damage_lower = 25
+	robust_searching = 1
+	vision_range = 10
+	aggro_vision_range = 21
+	minimum_distance = 1
+	stat_attack = CONSCIOUS | UNCONSCIOUS | DEAD
+	maxHealth = 450
+	health = 450
+	var/vored = list() //yes baldi vores the people he kills
```
Comment (NopemanMcHalt) [inline modular_skyrat/code/modules/mob/living/simple_animal/hostile/baldi.dm:25]: wtfdyjbutcl

Comment (BobJoga) [reply modular_skyrat/code/modules/mob/living/simple_animal/hostile/baldi.dm:25]: swagcat

**Comment by Owai-Seek** [comment]
This stuff is getting ridiculous. Why? Why would a living creature be inside a calculator, at like, all? Why would it be a bald math teacher with a ruler? This is just weird.

**Comment by ShadeAware** [comment]
Because its FUN and its FUNNY @Owai-Seek not everything needs to be explained.
Its just a funny gag for calculators.

**Comment by Commandersand** [comment]
@Owai-Seek why does a space station need a clown???
why does a space station need BDSM vendors???

**Comment by Akrilla** [comment]
@Owai-Seek This is already a thing with emagged arcade machines.

**Pull request closed without merging**